### ISSUE TITLE
run_test.py shouldn't require 'runipy

### ISSUE THREAD

**Issue by jmansour** [underworldcode/underworld2#100]
This script should be able to work without runipy (in which case it can't test ipynb files).

We should also add more tests/examples in pure python.

**Comment by jmansour** [comment]
This script should be able to work without runipy (in which case it can't test ipynb files).  
We should also add more tests/examples in pure python.

**Comment by lmoresi** [comment]
It would be good to try to write notebooks that can be exported to Python and be expected to run. That might constrain us a bit but would improve the workflow for people running large / parallel jobs.

They should run on multiple procs ... Should have said that !

Sent from my iPhone

On 3 Apr 2016, at 06:45, jmansour <notifications@github.com<mailto:notifications@github.com>> wrote:


This script should be able to work without runipy (in which case it can't test ipynb files).

We should also add more tests/examples in pure python.

**Comment by jmansour** [comment]
The runipy part of this is fixed.  e58298084704b7b24ea2d5a300d191260fb03cc3

I'll leave this open for now so that we can also address running tests in parallel.

**Comment by jmansour** [comment]
actually, started a new ticket #126
